common/src/main/java/tachyon/worker/DataServerMessage.java

```diff
@@ -36,8 +36,9 @@
 
   // The size of the prefix of the header: frame length (long), messageType (int)
   private static final int HEADER_PREFIX_LENGTH = 12;
-  // The request header is: HEADER_PREFIX, blockId (long), offset (long), length (long)
-  private static final int REQUEST_HEADER_LENGTH = HEADER_PREFIX_LENGTH + 24;
+  // The request header is: HEADER_PREFIX, blockId (long), offset (long), length (long),
+  // lockId (long), sessionId (long)
+  private static final int REQUEST_HEADER_LENGTH = HEADER_PREFIX_LENGTH + 40;
```

Comment:
Would it work to use something like `RPCBlockReadRequest.getEncodedSize()` instead of hardcoding the `+ 40`?